jupyter_book/book_template/content/guide/04_faq.md

```diff
@@ -78,3 +78,33 @@ Note that, if viewing the file on a Jupyter Notebook session, the figure will no
 
 If you've got questions, concerns, or suggestions, please open an issue at
 [at the jupyter book issues page](https://github.com/jupyter/jupyter-book/issues)
+
+## How should I add cell tags to my notebooks?
+
+You can control the behavior of Jupyter Book by putting custom tags
+in the metadata of your cells. But how do you actually do this?
+
+**Use the Jupyter Notebook cell tag editor**. The Jupyter Notebook ships with a
+cell tag editor by default. This lets you add cell tags to each cell quickly.
+
+To enable the cell tag editor, go click `View -> Cell Toolbar -> Tags`. This
+will enable the tags UI. Here's what the menu looks like.
+
+![](../images/tags_notebook.png)
+
+**Use the JupyterLab Cell Tags plugin**. JupyterLab is an IDE-like Jupyter
+environment that runs in your browser. It has a rich extension ecosystem.
+A particularly useful extension [is the "cell tags" plugin](https://github.com/jupyterlab/jupyterlab-celltags),
+which exposes a user interface that lets you quickly insert cell tags.
+
+You can install the cell tags plugin with the following command (after installing
+JupyterLab).
+
+```
+jupyter labextension install @jupyterlab/celltags
+```
+
+Then, you'll find tags under the "wrench" menu section.
+Here's what the tags UI in JupyterLab looks like.
+
+![](../images/tags_jupyterlab.png)
```

Comment:
maybe you should use tinypng to make the files smaller? Or is this as small as it gets?

Reply:
that's a good idea - will give this a shot